samples/TR/index.html

```diff
@@ -278,5 +273,40 @@
+	<tr>
+		<td>EPUB 3.1 Overview
+		<td><a href="//www.idpf.org/epub/31/spec/epub-overview-20160906.html">Classic</a>
+		<td><a href="../../viewer/vivliostyle-viewer.html#style=../samples/TR/W3C.css&amp;x=//cdn.rawgit.com/IDPF/epub-revision/20160906/build/31/spec/epub-overview.html">Vivliostyle</a>
+	<tr>
+		<td>EPUB Packages 3.1
+		<td><a href="//www.idpf.org/epub/31/spec/epub-packages-20160906.html">Classic</a>
+		<td><a href="../../viewer/vivliostyle-viewer.html#style=../samples/TR/W3C.css&amp;x=//cdn.rawgit.com/IDPF/epub-revision/20160906/build/31/spec/epub-packages.html">Vivliostyle</a>
+	<tr>
+		<td>EPUB Content Documents 3.1
+		<td><a href="//www.idpf.org/epub/31/spec/epub-contentdocs-20160906.html">Classic</a>
+		<td><a href="../../viewer/vivliostyle-viewer.html#style=../samples/TR/W3C.css&amp;x=//cdn.rawgit.com/IDPF/epub-revision/20160906/build/31/spec/epub-contentdocs.html">Vivliostyle</a>
+	<tr>
+		<td>EPUB Media Overlays 3.1
+		<td><a href="//www.idpf.org/epub/31/spec/epub-mediaoverlays-20160906.html">Classic</a>
+		<td><a href="../../viewer/vivliostyle-viewer.html#style=../samples/TR/W3C.css&amp;x=//cdn.rawgit.com/IDPF/epub-revision/20160906/build/31/spec/epub-mediaoverlays.html">Vivliostyle</a>
+	<tr>
+		<td>EPUB Open Container Format (OCF) 3.1
+		<td><a href="//www.idpf.org/epub/31/spec/epub-ocf-20160906.html">Classic</a>
+		<td><a href="../../viewer/vivliostyle-viewer.html#style=../samples/TR/W3C.css&amp;x=//cdn.rawgit.com/IDPF/epub-revision/20160906/build/31/spec/epub-ocf.html">Vivliostyle</a>
+	<tr>
+		<td>EPUB Accessibility 1.0
+		<td><a href="//www.idpf.org/epub/a11y/accessibility-20160906.html">Classic</a>
+		<td><a href="../../viewer/vivliostyle-viewer.html#style=../samples/TR/W3C.css&amp;x=//cdn.rawgit.com/IDPF/epub-revision/20160906/build/a11y/accessibility.html">Vivliostyle</a>
+	<tr>
+		<td>Alternate Style Tags 1.1
+		<td><a href="//www.idpf.org/epub/altss-tags/altss-tags-20160906.html">Classic</a>
+		<td><a href="../../viewer/vivliostyle-viewer.html#style=../samples/TR/W3C.css&amp;x=//cdn.rawgit.com/IDPF/epub-revision/20160906/build/altss-tags/altss-tags.html">Vivliostyle</a>
+	<tr>
+		<td>EPUB 3.1 Changes from EPUB 3.0.1
+		<td><a href="//www.idpf.org/epub/31/spec/epub-changes-20160906.html">Classic</a>
+		<td><a href="../../viewer/vivliostyle-viewer.html#style=../samples/TR/W3C.css&amp;x=//cdn.rawgit.com/IDPF/epub-revision/20160906/build/31/spec/epub-changes.html">Vivliostyle</a>
```

Comment:
It was difficult to find a specific document, e.g. "EPUB Content Documents 3.1", and I thought that separate versions are useful.